docs/how-to/setup-containerd-pull.md

```diff
@@ -0,0 +1,96 @@
+# Manual Steps for Setting Up Containerd and Docker for Genpolicy Tool
+
+This guide provides instructions for manually setting up your system to use the genpolicy tool with containerd and Docker. This setup is needed for managed identity-based authentication to private registries using an identity token.
+
+These are the steps performed by the script `src/tools/genpolicy/setup_containerd_docker.sh`
+
+## Prerequisites
+
+- Ensure you have `sudo` access on the system.
+- Determine the package manager used by your system (`apt-get` or `tdnf`).
+
+## Steps
+
+### 1. Install Containerd
+
```

Comment:
Add the OS version that you used for testing - because other versions might not work, and that's OK.